tests/test_dataset.py

```diff
@@ -29,11 +29,19 @@
 HYPOTHESIS_MAX_EXAMPLES = 20
 
 
-@pytest.fixture()
+@pytest.fixture
 def grid():
     return pyvista.UnstructuredGrid(examples.hexbeamfile)
 
 
+@pytest.fixture
+def clean_up_type_overrides():
+    """Reset type overrides on teardown."""
+    yield
+    for cls in pyvista._overrides.OVERRIDABLES:
```

Comment:
It's probably not nice to use a "private" module in the test suite, but I didn't want to expose the module into the public namespace. But the `from pyvista._overrides` import ensures that `pyvista._overrides` is actually there if anyone goes looking.